### ISSUE TITLE
Typescript Task Runner does not transpile and will not display any error in Output view

### ISSUE THREAD

**Issue by jayallanjethwa** [Microsoft/vscode#20946]
- VSCode Version: 1.9.1 (fresh Vanilla installation, no add-ons)
- OS Version: Windows 10 Version 1607 (OS Build 14393.693)

HelloWorld.ts
```javascript
class Startup {
    public static main(): number {
        console.log('Hello World');
        return 0;
    }
}

Startup.main();
```
tsconfig.json
```javascript
{
    "compilerOptions": {
        "target": "es5",
        "module": "commonjs",
        "sourceMap": true
    }
}
```

Steps to Reproduce:

1. Open folder containing above files in VSCode
2. Press Ctrl+Shift+P
3. Enter "configure"
4. Select "Configure Task Runner"
5. Select "Typescript - tsconfig.json"
6. Press Ctrl+Shift+B
7. Notice that Output view remains empty
8. Select "TypeScript" in filter dropdown in right section of Output view
9. Output: [Info  - 17:42:56] Using tsserver from location: c:\Program Files (x86)\Microsoft VS Code\resources\app\extensions\typescript\node_modules\typescript\lib\tsserver.js
10. Notice that no error is displayed and no HelloWorld.js is created

Expected behavior:

Pressing Ctrl+Shift+B transpiles HelloWorld.ts to HelloWorld.js or at least outputs an error message in the Output view of VSCode.

**Comment by dbaeumer** [comment]
This works for me on latest without any problems. Do you have tsc installed using npm install -g typescript

**Comment by jayallanjethwa** [comment]
Fixed with 1.10.2, Output window now correctly displays:
"Failed to launch external program 'tsc'"
